taxcalc/current_law_policy.json

```diff
@@ -1124,6 +1124,28 @@
         "out_of_range_maxmsg": "",
         "out_of_range_action": "stop"
     },
+        
+    "_ID_StateLocalTax_crt": {
+        "long_name": "Maximum state and local income and sales taxes deduction as a fraction of AGI.",
```

Comment:
Could you take a look at `_ID_Charity_crt_all` (https://github.com/open-source-economics/Tax-Calculator/blob/master/taxcalc/current_law_policy.json#L1262) and model your `long_name` and `description` off of that?

Reply:
I'll be sure to model the ID_StateLocalTax_ct description after that parameter. Thanks!